voice/transcription.go

```diff
@@ -92,3 +92,16 @@ func (trans *Transcription) Contents(client *messagebird.Client) (string, error)
 	b, err := ioutil.ReadAll(resp.Body)
 	return string(b), err
 }
+
+// CreateTranscription creates a transcription request for an existing recording
+func CreateTranscription(client *messagebird.Client, callID string, legID string, recordingID string) (trans *Transcription, err error) {
+	var body struct{}
+	path := fmt.Sprintf("/calls/%s/legs/%s/recordings/%s/transcriptions", callID, legID, recordingID)
+	var resp struct {
+		Data []Transcription `json:"data"`
+	}
+	if err := client.Request(&resp, http.MethodPost, apiRoot+path, body); err != nil {
```

Comment:
This would have`{}` as json body. If that is not the expected behaviour use `nil`

Reply:
API doesn't respond well to `nil` with current client code: https://github.com/messagebird/go-rest-api/pull/66/files/c24e2114d94674a74e43f7d5d2773866f70027d1#r311526617